index.bs

```diff
@@ -273,73 +273,68 @@ requestFontsButton.onclick = async function() {
 # Concepts # {#concepts}
 <!-- ============================================================ -->
 
-Issue: Define any new concepts beyond just the API
+The <dfn>user language</dfn> is a valid BCP 47 language tag representing either [=/a plausible language=] or the user's most preferred language. [[!BCP47]]
 
 <!-- ============================================================ -->
 ## Font Representation ## {#concept-font-representation}
 <!-- ============================================================ -->
 
-A <dfn>font representation</dfn> is an OpenType [[!OPENTYPE]] definition of a font. Even if the font was originally described in another file format, it is assumed that if it is supported by a user agent then an OpenType representation can be derived for it. This includes True Type [[TrueType]], Web Open Font Format 1.0 [[WOFF]] and Web Open Font Format 2.0 [[WOFF2]] files.
+A <dfn>font representation</dfn> is some concrete representation of a font. Examples include OpenType [[!OPENTYPE]], TrueType [[!TRUETYPE]], bitmap fonts, Type1 fonts, SVG fonts, and future font formats. This specification defines the properties of a [=/font representation=] as:
 
 <div dfn-for="font representation">
 
-A [=/font representation=] is serialized in SFNT [[!SFNT]] format, a flexible and extensible tabled-based container format which can contain font data in a multitude of other formats.
-
-The <dfn>data bytes</dfn> of a [=/font representation=] is the SFNT [[!SFNT]] serialization of the representation, which is a [=/byte sequence=] encoding a <dfn>table list</dfn>, a [=/list=] of [=/font tables=].
+* The <dfn>data bytes</dfn>, which is a [=/byte sequence=] containing a serialization of the font.
+* The <dfn>PostScript name</dfn>, which is a {{DOMString}}. This is commonly used as a unique identifier for the font during font loading, e.g. "Optima-Bold". This must be valid as defined in [[!RFC8081]].
```

Comment:
RFC 8081 doesn't appear to define the valid characters in a PostScript name, just in fragment identifiers referring to them. [Section 4.2](https://www.rfc-editor.org/rfc/rfc8081#section-4.2) quotes [ISO.14496-22.2015] as doing so, so the RFC a good free reference to the character set to put in a note, but the normative reference should probably be to ISO 14496-22. Amusingly, the RFC's reference is now a 404, replaced by https://standards.iso.org/ittf/PubliclyAvailableStandards/c074461_ISO_IEC_14496-22_2019.zip. https://www.iso.org/standard/74461.html might be more stable.

Or it might make sense to copy the definition here, since if they expand the valid set, we still might want to filter them out for web compatibility, and if they shrink the valid set, the newly-invalid characters will probably still appear in this API.

Reply:
Made the definition here, using infra parlance.